conversion.go

```diff
@@ -0,0 +1,32 @@
+package seccomp // import "github.com/seccomp/containers-golang"
+
+import "fmt"
+
+var goArchToSeccompArchMap = map[string]Arch{
```

Comment:
Should be safe for multiple readers and no writers in parallel.

Reply:
Yes, that's safe. Would be nice to have const maps in Go.